### DISCUSSION TITLE
Wait_Start for all containers

### DISCUSSION THREAD

**Discussion by lmb2100** [Security-Onion-Solutions/securityonion#10795] [General]
I have a distributed deployment and I did a so-nsm-clear and so-elastic-clear on a forward node and now it says all of the containers on the sensor is wait_start. When I try to start the containers indiviually it says "Rendering SLS 'minions.securityonion2_sensor'  failed. Please see master log for details." It was working before that command so I am not sure what happened because I did it on another sensor and it is working fine. I dont even know where the master log is. Thank you!!

**Comment by lmb2100** [comment]
Please see attached for Screenshots. Also is there a way to delete logs from the SOC? After I did those commands the logs still showed 50 Million+ on the SOC. I just wanted to get rid of the logs that were generated from testing.
![port groups on manager](https://github.com/Security-Onion-Solutions/securityonion/assets/134076799/64fad98c-8d7e-494c-bd8c-d8118639ca34)
![sal-minion -l debug](https://github.com/Security-Onion-Solutions/securityonion/assets/134076799/86c9370e-b270-428e-9361-d15ec988502a)
![salt issues](https://github.com/Security-Onion-Solutions/securityonion/assets/134076799/ae71ea49-b777-4429-a8c7-0078686ce335)
![Netstat](https://github.com/Security-Onion-Solutions/securityonion/assets/134076799/93c11a18-f299-45ed-b850-c028ad3abb82)

**Reply by lmb2100** [reply]
![etc-minion and error](https://github.com/Security-Onion-Solutions/securityonion/assets/134076799/512288c3-8453-4d1c-be97-fa08c02f8c2f)

**Comment by cm-ops** [comment]
You would need to run `salt-call` from the minion, the salt master service is only running on your manager node. 

The master log is at `/opt/so/log/salt/master` on your manager node. 

What does your minion pillar file look like at `/opt/so/saltstack/local/pillar/minions/securityonion2_sensor.sls`?

**Reply by lmb2100** [reply]
What else in the command salt-call? I am pretty new to salt and SO so please bear with me. Please see attached for the pillar file for the minion. Thanks!
![Pillar File](https://github.com/Security-Onion-Solutions/securityonion/assets/134076799/e8130648-12fb-4ff8-b92a-692f1a7c0f91)

**Comment by lmb2100** [comment]
So am I running a masterless minion? The salt master service is running on the manager but neither can find the master. The manager can still see one of the nodes on the grid and running salt-run manage.up shows the sensors. Thanks again!

**Comment by cm-ops** [comment]
You have to fix your firewall pillar:

```
firewall:
  assigned_hostgroups:
    chain:
      DOCKER-USER:
        hostgroups:
          wazuh_agent:
            portgroups:
              - portgroups.wazuh_agent
          syslog:
            portgroups:
              - portgroups.syslog
```
Pay attention to the spacing in the yaml file.
